packages/aws-helpers/src/stagedRollouts/utils/log.ts

```diff
@@ -0,0 +1,7 @@
+export function logDebug(...args: any[]) {
+    // If DEBUG variable is not set during build,
+    // this function should be minified to an empty one and removed completely from the bundle.
+    if (process.env.DEBUG === "true") {
```

Comment:
Yeah, I wasn't sure about that. Logging these kind of errors adds really not much, because it's very difficult to find anything in CloudWatch if you don't know what are you looking for.
For now, because I didn't know what are best practices we use for logging, I added some logs, that helped me debug while developing.